README.md

```diff
@@ -32,8 +32,9 @@ Incubated API Repository to evolve and maintain the definitions and documentatio
 ## Scope
 
 * Service API(s) “QoSBooking” (see APIBacklog.md) 
-* The API(s) provide(s) the customer with the ability to:  
-  * §Short API scope description§ !! Add here a scope description according to the API proposal
+* The API(s) provide(s) the API consumer with the ability to:  
```

Comment:
```suggestion
* The API provides the API consumer with the ability to:  
```

There is currently only a single API. The suggestion simplifies the reading of the sentence.